src/components/Nav/Nav.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useEffect } from "react";
+import "./Nav.css";
+import {MdOndemandVideo} from "react-icons/md"
+import { NavLink } from "react-router-dom";
+
+function Nav() {
+  const [show, handleShow] = useState(false);
+
+  useEffect(() => {
+    window.addEventListener("scroll", () => {
```

Comment:
you are using vanilla js in react which is not a good practice you can use react event as onScroll here.